src/blocks/DetailsBlock/DetailsBlock.astro

```diff
@@ -0,0 +1,45 @@
+---
+import type { DetailsBlockFragment } from '@lib/types/datocms';
+
+interface Props {
+  block: DetailsBlockFragment
+}
+
+const { block } = Astro.props;
+const splitChar = '|';
+const hasMultipleValues = (value: string) => value.includes(splitChar);
+const splitValues = (value: string) => value.split(splitChar);
+---
+
+<dl class:list={['details-block', `details-block--${block.display}`]}>
+  {
+    Object.keys(block.details).map((key) => (
+      <>
+        <dt>{key}</dt>
+        {hasMultipleValues(block.details[key]) ? (
+          <div>
```

Comment:
Is this valid HTML? I didn't know that. I still think you might be able to do without the `<div>`. See https://gist.github.com/LeaVerou/12b1d1e078510900c7ab206da1a0ff2e?permalink_comment_id=4537764#gistcomment-4537764

Reply:
You're right. I thought it was valid, but I've looked it up now and found out it is not valid. I can fix it without a div.